kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/marketplace/model/NativeOffer.java

```diff
@@ -0,0 +1,12 @@
+package com.kin.ecosystem.marketplace.model;
+
+import com.kin.ecosystem.network.model.Offer;
+
+
+public class NativeOffer extends Offer {
```

Comment:
NativeOffer shouldn't inherit from com.kin.ecosystem.network.model.Offer

Reply:
why not? at the end, it should be same as the list type